packages/canary-web-components/src/components/ic-pagination-bar/ic-pagination-bar.tsx

```diff
@@ -160,7 +170,7 @@ export class PaginationBar {
   /**
    * Emitted when a page is navigated to via the 'go to' input.
    */
-  @Event() icPageChange: EventEmitter<{ value: number }>;
+  @Event() icPageChange: EventEmitter<IcPageChangeEventDetail>;
```

Comment:
Would it be worth adding a sentence to the event description to explain about what the `fromItemsPerPage` flag is for?